apps/project/schema.py

```diff
@@ -415,6 +418,7 @@ class Meta:
         )
 
     analysis_framework = graphene.Field(AnalysisFrameworkDetailType)
+    assessment_registries = graphene.List(graphene.NonNull(AssessmentRegistryType))
```

Comment:
Let's define this in assesssment_registry/schema.py and then use it as others. Look at LeadQuery for reference.

Also, please use pagination.